colabs/Pierwsze_wnioski_z_data_setu.ipynb

```diff
@@ -0,0 +1,2689 @@
+              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
+              "      fill: #FFFFFF;\n",
+              "    }\n",
+              "  </style>\n",
+              "\n",
+              "      <script>\n",
+              "        const buttonEl =\n",
+              "          document.querySelector('#df-70ec6439-1cbb-4b9d-96af-fc6c97ffd2c6 button.colab-df-convert');\n",
+              "        buttonEl.style.display =\n",
+              "          google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
+              "\n",
+              "        async function convertToInteractive(key) {\n",
+              "          const element = document.querySelector('#df-70ec6439-1cbb-4b9d-96af-fc6c97ffd2c6');\n",
+              "          const dataTable =\n",
+              "            await google.colab.kernel.invokeFunction('convertToInteractive',\n",
+              "                                                     [key], {});\n",
+              "          if (!dataTable) return;\n",
+              "\n",
+              "          const docLinkHtml = 'Like what you see? Visit the ' +\n",
+              "            '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
+              "            + ' to learn more about interactive tables.';\n",
+              "          element.innerHTML = '';\n",
+              "          dataTable['output_type'] = 'display_data';\n",
+              "          await google.colab.output.renderOutput(dataTable, element);\n",
+              "          const docLink = document.createElement('div');\n",
+              "          docLink.innerHTML = docLinkHtml;\n",
+              "          element.appendChild(docLink);\n",
+              "        }\n",
+              "      </script>\n",
+              "    </div>\n",
+              "  </div>\n",
+              "  "
+            ]
+          },
+          "metadata": {}
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Wykresy"
+      ],
+      "metadata": {
+        "id": "HuVRqAug4WRU"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Poniższe wykresy i tabele mogą okazać się pomocne w celu np. zbadania zbilansowania danych"
+      ],
+      "metadata": {
+        "id": "ALp5XABMn_Gh"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "#### Rok produkcji"
+      ],
+      "metadata": {
+        "id": "D6gb4j0v9eXr"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "data_start_year_count = title_basics.groupBy(\"startYear\").count()\n",
+        "data_start_year_count_pd = data_start_year_count.toPandas()\n",
+        "\n",
+        "data_start_year_count_pd = data_start_year_count_pd.sort_values(by = [\"startYear\"])\n",
+        "data_start_year_count_pd = data_start_year_count_pd[1:(len(data_start_year_count_pd) - 8)]\n",
```

Comment:
ta 8 to tak zwany antywzorzec liczby migicznej. Nie mówiąc, że 1: powoduje odrzucenie rekordu 0 z nieznanych mi powodów.
[o antywzorcu](https://codeburst.io/software-anti-patterns-magic-numbers-7bc484f40544)

Reply:
poprawione - odrzucenie lat 2022: wykonane za pomocą where